image/image.css

```diff
@@ -0,0 +1,13 @@
+.img {
+    max-width: 34rem;

```

Comment:
Why have you given this specific height and width?

Reply:
If I will not give then it will cover all the available width & height space. By giving this I have defined a particular range.

Reply:
Is it not hampering image's responsiveness?

Reply:
Okay